source/compatibility.tex

```diff
@@ -64,6 +64,30 @@
                                 // changed according to the global C locale
 \end{codeblock}
 
+\rSec2[diff.cpp23.depr]{\ref{depr}: compatibility features}
+
+\nodiffref
+\change
+Remove header \tcode{<codecvt>} and all its contents.
```

Comment:
Do we maybe want to retain an index entry for the mention of the header name?
If someone is looking for that header in a newer standard, the index should point here.
-> use `\libnoheader`

Reply:
Done!